tools/flub/README.md

```diff
@@ -0,0 +1,121 @@
+# @fluid-internal/flub
+
+flub is a build and release tool for the Fluid Framework GitHub repositories. flub is intended to replace the existing
```

Comment:
Something I have been recommending for github-flavored markdown is line-breaking on sentence boundaries. So long as the lines are adjacent, they will be rendered in the same para. Line breaking on sentence boundaries then makes future changes scope a bit better.

Reply:
I used to agree, but I stopped because the MSDocs guidance is the opposite, and after reviewing a lot of changes with line breaks, diffs are waaaay easier to read when everything is on a single line, because line break changes are always meaningful.

Reply:
Actually, re-reading your comment, I think I misunderstood. You're advocating a line break after every sentence, right? I completely misunderstood, and I'm willing to try that. Do you know of any VSCode extensions to automate it, sort of like  the [rewrap extension](https://stkb.github.io/Rewrap/)?

Reply:
One potential issue is that some of our markdown files get processed by automated tools, and those tools may interfere with our policies. In most cases I can think of, though, the tools generally don't format anything that's been manually formatted.

Reply:
I hadn't seen the rewrap extension before. That's cool - I'm going to have to give that a shot.

I haven't looked into extensions for doing sentence breaks before, but I'll have to do a search. Generally, I just use that pattern when writing new docs or editing existing docs. But I see this as just a sort of nice-to-have sort of policy, rather than being particularly important. So if our tools disagree, I'm not concerned.

Also, definitely don't let this block your PR. This is just something I've been low-key advocating for 🙂